### PR TITLE
Update to Postgres 13

### PR DESCRIPTION
**Pull request by jimman2003**
feature
### Technical

A new enviroment variable is needed ,because of the new image
### Testing

Tested edits on works,editions,lists,reading logs (add and remove)
Tested author merges
Tested login as admin/librarian

### Stakeholders

@mekarpeles @cdrini

### CODE REVIEW

**Comment by cdrini** [comment]
Nice! Can you explicitly list your testing steps? That'll help us add any test cases that you might've missed :)

File: docker-compose.override.yml [hunk 1]
```diff
@@ -39,9 +39,11 @@ services:
       - .:/openlibrary
 
   db:
-    image: postgres:9.3
+    image: postgres:13.4
     networks:
       - dbnet
+    environment:
+      - POSTGRES_HOST_AUTH_METHOD=trust
```
Comment (cdrini) [inline docker-compose.override.yml:46]: Can you add a comment about what this does/why it's necessary?

Comment (jimman2003) [reply docker-compose.override.yml:46]: Well , it is a requirement for the new image

**Comment by jimman2003** [comment]
I took the lazy approach, just browsed to the categories that has books in the dump :)

**Review by mekarpeles — CHANGES_REQUESTED** [review]

**Comment by mekarpeles** [comment]
This now lgtm, ty!

**Pull request merged**